test.sh

```diff
@@ -236,6 +238,17 @@ clean_up()
     fi
 }
 
+convert_notebooks()
+{
+    echo "testing notebooks"
+    for notebook in `grep ipynb docs/tutorials.rst | sed -e 's/^[ \t]*//'`
+    do
+        jupyter nbconvert --to notebook --execute "docs/$notebook"
```

Comment:
What's the point of setting a limit?

Reply:
That flag doesn't do anything for me when I do:

```
upyter nbconvert --to notebook --ExecutePreprocessor.timeout=10 --execute "docs/Tutorial_Matrix_Profiles_For_Streaming_Data.ipynb"
```

Reply:
@seanlaw 

> What's the point of setting a limit?

Just to make sure the cells have enough time to be executed. We can set to `-1` (no limit). 

> That flag doesn't do anything for me when I do

Thank you for checking it. So, it may not be effecitve (?) I am going to ignore this flag and leave it as is.